Common/License.txt

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
The copyright headers refers to the license file in the project root, so I've added licenses to the projects that didn't have them (same the Scanner for MSBuild).